hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/BackgroundPipelineCreator.java

```diff
@@ -96,7 +96,7 @@ private void createPipelines() {
           if (scheduler.isClosed()) {
             break;
           }
-
+          pipelineManager.scrubPipeline(type, factor);
```

Comment:
move the scrubPipeline out of the while loop since it will scrub all stale pipline in one call.

Reply:
Ok.